decidim-assemblies/app/commands/decidim/assemblies/admin/update_assemblies_setting.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Assemblies
+    module Admin
+      # A command with all the business logic when updating assemblies
+      # settings in the system.
```

Comment:
As this command is performed from the `/admin`, not the `/system`, I suggest to change the doc to admin:
```suggestion
      # settings in the admin area.
```